layouts/partials/javascript.html

```diff
@@ -5,3 +5,4 @@
 <script src="https://cdn.jsdelivr.net/npm/ua-parser-js@0.7.25/src/ua-parser.js" integrity="sha256-zkQSSL5rBWlvCbIoTPQikbYHhOeeiTJy9YX9LXLvFBk=" crossorigin="anonymous"></script>
 <script src="https://cdn.jsdelivr.net/npm/lunr@2.3.9/lunr.min.js" integrity="sha256-DFDZACuFeAqEKv/7Vnu1Tt5ALa58bcWZegGGFNgET8g=" crossorigin="anonymous"></script>
 <script src="{{ $app.RelPermalink }}" type="module" integrity="{{ $app.Data.Integrity }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js" integrity="sha384-Wuix6BuhrWbjDBs24bXrjf4ZQ5aFeFWBuKkFekO2t8xFU0iNaLQfp2K6/1Nxveei" crossorigin="anonymous"></script>
```

Comment:
Is there a reason not to use [KaTeX](https://katex.org/)? This has been the recommended JS renderer for web from both @hultonha and @jeremyong-az .

Reply:
I have no horse in this race and know nothing about KaTeX or TeXML 😅 @jeremyong-az is definitely the expert 😝 I'll defer to his judgement